meteor/client/ui/Settings/ShowStyle/VariantSettings.tsx

```diff
@@ -159,13 +266,27 @@ export const ShowStyleVariantsSettings = withTranslation()(
 			return (
 				<div>
 					<h2 className="mhn">{t('Variants')}</h2>
+					<div className="mod mhs"></div>
 					<table className="table expando settings-studio-showStyleVariants-table">
 						<tbody>{this.renderShowStyleVariants()}</tbody>
 					</table>
 					<div className="mod mhs">
 						<button className="btn btn-primary" onClick={this.onAddShowStyleVariant}>
 							<FontAwesomeIcon icon={faPlus} />
 						</button>
+						<UploadButton
+							className="btn btn-secondary mls"
+							accept="application/json,.json"
+							onChange={(e) => this.importVariants(e)}
```

Comment:
It's a button, what's the reason for not using the `onClick` event?

Reply:
The custom UploadButton has been used in this manner everywhere else, it is designed with that specific purpose.